Dockerfile

```diff
@@ -19,9 +19,13 @@ FROM tomcat:9
 
 RUN apt-get update && apt-get upgrade -y
 
-RUN wget -O dd-java-agent.jar https://dtdg.co/latest-java-tracer
+RUN wget -O dd-java-agent.jar https://dtdg.co/latest-java-tracer && \
```

Comment:
Consider adding a comment with a link to the documentation where you're getting this from